cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -282,16 +279,24 @@ function Get_ContainerVersionLabel {
 
 # Builds the pause image with name $INFRA_CONTAINER.
 function Create-PauseImage {
-  $version_label = Get_ContainerVersionLabel `
-      $(Get-InstanceMetadataValue 'win-version')
+  $win_version = $(Get-InstanceMetadataValue 'win-version')
+  if ($win_version -match '2019') {
+    # TODO(pjh): update this function to properly support 2019 vs. 1809 vs.
+    # future Windows versions. For example, Windows Server 2019 does not
+    # support the nanoserver container
+    # (https://blogs.technet.microsoft.com/virtualization/2018/11/13/windows-server-2019-now-available/).
```

Comment:
Does that mean we'll have to use 1809 nanoserver containers on 2019 hosts?

Reply:
yes @yujuhong  that's one of the options. you can use either of the 3 following container images on Server 2019
1. servercore:1809
2. nanoserver:1809
3. servercore:ltsc2019